cmd/wallet-web/src/components/FlyoutMenu/FlyoutMenu.vue

```diff
@@ -48,32 +48,48 @@
         <tool-tip :tool-tip-label="t('CredentialDetails.toolTipLabel')"></tool-tip>
       </div>
     </button>
-    <div v-if="showFlyoutMenuList" id="flyoutMenuList" class="relative">
-      <flyout-menu-list :credential-id="credentialId" />
+    <button
```

Comment:
On line https://github.com/trustbloc/wallet/pull/1285/files#diff-74582897998985f82cb6c1a6ad7463dde548239548e7f67e729ea87d518a3db3R36, change to the following:
```js
@focus="showTooltip = false"
```

Reply:
On line https://github.com/trustbloc/wallet/pull/1285/files#diff-74582897998985f82cb6c1a6ad7463dde548239548e7f67e729ea87d518a3db3R44, change to the following:
```js
@mouseover="showTooltip = true"
```
And on the following line, also add `@mouseout` event handler to set the `showTooltip` to `false`.

Reply:
On line https://github.com/trustbloc/wallet/pull/1285/files#diff-74582897998985f82cb6c1a6ad7463dde548239548e7f67e729ea87d518a3db3R47, move `v-if` to the `tooltip` directly, remove `id`, and get rid of the `div`

Reply:
updated as commented